charts/openbao/Chart.yaml

```diff
@@ -26,6 +26,9 @@ annotations:
   charts.openshift.io/name: Openbao
   artifacthub.io/containsSecurityUpdates: "true"
   artifacthub.io/changes: |
+    - kind: fixed
+      description: |
+        fix: grafana dashboard
     - kind: changed
```

Comment:
Can you remove the old annotations? The annotations should alway reflect only the oldes change. Thx.